package.json

```diff
@@ -81,7 +80,9 @@
     "babel-plugin-transform-runtime": "6.23.0",
     "babel-polyfill": "6.26.0",
     "babel-preset-env": "1.6.1",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
J'ai hélas dû ajouter ces 2 deps babel pour que jest accepte d'exécuter mes tests... Et je ne sais pas trop pourquoi :/

Reply:
Il se passait quoi sans ca ? Il ne transformait pas les fichier js/jsx ?

Reply:
Non, j'avais une `Couldn't find preset "es2015" relative to directory <path>/node_modules/identity-obj-proxy` ...